src/target/ext/host_io.rs

```diff
@@ -24,6 +24,18 @@ bitflags! {
         const O_TRUNC = 0x400;
         /// Exclusive access.
         const O_EXCL = 0x800;
+
+        // LLDB-specific flags, as documented at:
+        // https://github.com/llvm/llvm-project/blob/ec642ceebc1aacc8b16249df7734b8cf90ae2963/lldb/include/lldb/Host/File.h#L47-L66
+
+        /// Do not block
```

Comment:
Can you prefix these doc comments with `(LLDB extension)`?

Oh, and end each comment with a `.` (for consistency)